### PR TITLE
Update parameters

### PR DESCRIPTION
**Pull request by kylacochrane**
This PR updates various `profile_dists` and `gas_call` parameters and adjusts their corresponding defaults based on requirements of the gasnomenclature pipeline. 
It adds additional definitions and pattern requirements to the nextflow_schema.json. 

Documentation for the pipeline has also been updated. 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in docs/usage.md is updated.
- [x] Output Documentation in docs/output.md is updated.
- [x] README.md is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md
Comment (emarinier) [inline README.md:31]: I think I remember this coming up at one point, but if there's only one option, why the parameter?

File: README.md
Comment (emarinier) [inline README.md:38]: I think for clarity the first sentence could include "file path", maybe something like: "Path to a file that defines loci to keep ..."

File: modules/local/cluster_file/main.nf
Comment (emarinier) [inline modules/local/cluster_file/main.nf:3]: I think this is similar but slightly different to the module in #11. Maybe the header generation stuff? Not sure if it'll cause a conflict later when one of them is merged.

**Review by emarinier — CHANGES_REQUESTED** [review]
Looks really good. Just minor suggestions.

File: README.md
Comment (apetkau) [reply README.md:31]: That's a good point. We can remove the parameter in another PR. I had kept `--pd_outfmt` in the gasclustering pipeline for now in case there was changes to profile_dists/gas, but probably best to remove the `--pd_outfmt` parameter until that is the case.

File: docs/usage.md
Comment (apetkau) [inline docs/usage.md:32]: I would remove the "de novo clustering" part, since it's running `gas call`, which is assigning samples to existing clusters (`gas mcluster` is what is performing de novo clustering).

File: modules/local/cluster_file/main.nf
Comment (apetkau) [reply modules/local/cluster_file/main.nf:3]: Since PR #11 hasn't been merged in yet, this PR contains some of the code from that PR, which should disappear once #11 is merged into `dev` (and `dev` is merged into this branch).

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks great. A few comments. Once PR #11 is merged into `dev`, can you merge `dev` into this branch, which should remove some of the extra changes displayed here?

File: modules/local/cluster_file/main.nf
Comment (kylacochrane) [reply modules/local/cluster_file/main.nf:3]: Thanks Aaron 😄 I will be sure to merge https://github.com/phac-nml/gasnomenclature/pull/11 to `dev` first and reverse merge `dev` back here.

File: README.md
Comment (kylacochrane) [reply README.md]: Agreed! Clarity added here: [a1b3685](https://github.com/phac-nml/gasnomenclature/pull/12/commits/a1b36852f018497b8fd50144b7031a1ae7f97876)

File: docs/usage.md
Comment (kylacochrane) [reply docs/usage.md]: Fixed here: [a1b3685](https://github.com/phac-nml/gasnomenclature/pull/12/commits/a1b36852f018497b8fd50144b7031a1ae7f97876)

File: README.md
Comment (kylacochrane) [reply README.md]: `--pd_outfmt` parameter has been removed here: [04ee4fd](https://github.com/phac-nml/gasnomenclature/pull/12/commits/04ee4fd8044719d6d76a2c786266337af7d829b3)

And the README was updated here: [a1b3685](https://github.com/phac-nml/gasnomenclature/pull/12/commits/a1b36852f018497b8fd50144b7031a1ae7f97876)

**Review by apetkau — APPROVED** [review]
Everything looks good to me. Good to merge after PR #11

**Review by emarinier — APPROVED** [review]

**Pull request merged**
